qvm/sys-firewall.sls

```diff
@@ -16,21 +16,29 @@
 #   qubesctl state.sls qvm.sys-firewall dom0
 ##
 
+{% set default_template = salt['cmd.shell']('qubes-prefs default-template') %}
+
 include:
+  - qvm.default-dispvm
   - qvm.sys-net
 
 {%- from "qvm/template.jinja" import load -%}
 
 {% load_yaml as defaults -%}
 name:          sys-firewall
 present:
+  {% if salt['pillar.get']('qvm:sys-firewall:disposable', false) %}
+  - class:     DispVM
+  - template:  {{default_template}}-dvm
+  {% endif %}
   - label:     green
 prefs:
   - netvm:     sys-net
   - autostart: true
   - provides-network: true
   - memory:       500
 require:
+  - qvm:       {{default_template}}-dvm
```

Comment:
This (and in other places) dependency should be added only in case of disposable sys-* VMs.